pkg/api/http.go

```diff
@@ -33,27 +35,6 @@ func versionMiddleware(next http.Handler) http.Handler {
 	})
 }
 
-// TODO: use Istio tracing package
-// https://github.com/istio/istio/blob/master/pkg/tracing/config.go
-func copyTracingHeaders(from *http.Request, to *http.Request) {
```

Comment:
Can you please add a unit test for `api/echo` and prove that the OTEL client does forward all these headers?